mypy/stubtest.py

```diff
@@ -1353,11 +1353,16 @@ def anytype() -> mypy.types.AnyType:
         )
 
     # Try and look up a stub for the runtime object
-    stub = get_stub(type(runtime).__module__)
+    if isinstance(runtime, type):
+        runtime_type = runtime  # This might be a class
+    else:
+        runtime_type = type(runtime)  # Or an instance
+
+    stub = get_stub(runtime_type.__module__)
```

Comment:
It might be worth being a bit more defensive here as well. What if it's a class that doesn't have a `__module__` attribute, or that has a `__module__` attribute set to something that isn't a `str`? (The problem predates this PR, but we may as well fix it while we're here.)